src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        Colors randomColor = ColorSupplier.getRandomColor();
+        int randomNumber = new Random().nextInt(100);
```

Comment:
avoid magic numbers

create ColorSupplier and Random class level variables - to avoid creating a new instance each time this method is invoked